SpatialGDK/Source/SpatialGDKEditor/Private/SpatialTestSettings.cpp

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma optimize("", on)
```

Comment:
left in pragma tags, also they are the wrong way around?

Reply:
Or is this a way of optimizing hot code when running a debug build?

Reply:
Brain overload yesterday so accidentally left in and back to front!

Reply:
Happens : )